nano/core_test/ledger_walker.cpp

```diff
@@ -50,7 +50,7 @@ TEST (ledger_walker, genesis_account_longer)
 
 	nano::ledger_walker ledger_walker{ node->ledger };
 	EXPECT_TRUE (ledger_walker.walked_blocks.empty ());
-	EXPECT_EQ (1, ledger_walker.walked_blocks.bucket_count ());
```

Comment:
Would it be better to remove that line alltogether if it not directly relevant?

Reply:
It doesn't hurt either. It somehow proves that the hash map is a fresh one instead of just having been emptied (but memory not entirely released).